docker-compose-cypress.yml

```diff
@@ -0,0 +1,22 @@
+version: '2.1'
+
+services:
+  cypress:
+    image: carbon_cypress
+    container_name: cypress
+    command: bash -c 'cypress run'
+    environment:
+      cypress_localhost: http://storybook:9001/
+    depends_on:
+      - storybook
+    # Reason for the below `ipc: host` line:
+    # https://github.com/cypress-io/cypress/issues/350
+    # https://docs.cypress.io/guides/references/error-messages.html#The-Chromium-Renderer-process-just-crashed
+    ipc: host
+    # volumes:
```

Comment:
If the volumes setup can these 2 lines be removed?

Reply:
So that's really useful for when running this locally, which is why i left it in but commented out. We may also look at getting CI to upload these screenshots to S3 or something in future.